tcms/rpc/api/testexecution.py

```diff
@@ -107,7 +107,7 @@ def create(values):
             >>> TestExecution.create(values)
     """
 
-    form = NewExecutionForm(values)
+    form = NewTestExecutionForm(values)
 
     if not isinstance(values, dict):
         raise TypeError('Argument values must be in dict type.')
```

Comment:
The block where a new execution is created should be refactored as well. See testcase.py create() and update() functions.